--- atividades/0104GEO001/0104GEO00102.js	2012-11-09 16:07:33.794163458 -0200
+++ atividades/0104GEO001/0104GEO00102.js.new	2012-11-09 16:07:33.794163458 -0200
@@ -424,7 +424,7 @@
 
 var CorrectIndicator = ':-)';
 var IncorrectIndicator = 'X';
-var YourScoreIs = '<br>A sua pontua&#x00E7;&#x00E3;o &#x00E9;: ';
+var YourScoreIs = '<br>A sua pontua&ccedil;&atilde;o &eacute;: ';
 var CorrectResponse = 'MUITO BEM!';
 var IncorrectResponse = 'TENTE NOVAMENTE';
 var TotalUnfixedLeftItems = 0;
