--- atividades/0104GEO001/0104GEO00105.js	2012-10-02 09:57:59.071845350 -0300
+++ atividades/0104GEO001/0104GEO00105.js.new	2012-10-02 10:12:57.447858849 -0300
@@ -445,7 +445,7 @@
 
 
 
-	PreloadImages('../imagens/logo.png','Mato_Grosso.gif','Mato_Grosso_do_Sul.gif','Goias.gif','rodape.png');
+	PreloadImages('../imagens/logo.png','mato_grosso.gif','mato_grosso_do_sul.gif','goias.gif','rodape.png');
 
 
 	SetUpItems(ShuffleQs,QsToShow);
