--- atividades/0104GEO001/0104GEO00107.js	2012-10-02 09:57:59.095845351 -0300
+++ atividades/0104GEO001/0104GEO00107.js.new	2012-10-02 10:12:58.907858871 -0300
@@ -445,7 +445,7 @@
 
 
 
-	PreloadImages('../imagens/logo.png','Parana.gif','Santa_Catarina.gif','Rio_Grande_do_Sul.gif','rodape.png');
+	PreloadImages('../imagens/logo.png','parana.gif','santa_catarina.gif','rio_grande_do_sul.gif','rodape.png');
 
 
 	SetUpItems(ShuffleQs,QsToShow);
