--- atividades/0104GEO001/0104GEO00105.js	2012-11-09 15:52:03.242149475 -0200
+++ atividades/0104GEO001/0104GEO00105.js.new	2012-11-09 16:07:35.766163488 -0200
@@ -445,7 +445,7 @@
 
 
 
-	PreloadImages('../imagens/logo.png','Mato_Grosso.gif','Mato_Grosso_do_Sul.gif','Goias.gif','rodape.png');
+	PreloadImages('../imagens/logo.png','mato_grosso.gif','mato_grosso_do_sul.gif','goias.gif','rodape.png');
 
 
 	SetUpItems(ShuffleQs,QsToShow);
