--- atividades/0104GEO001/0104GEO00102.js	2012-11-09 15:52:03.250149476 -0200
+++ atividades/0104GEO001/0104GEO00102.js.new	2012-11-09 16:07:33.550163453 -0200
@@ -445,7 +445,7 @@
 
 
 
-	PreloadImages('../imagens/logo.png','Tocantins.gif','Rondonia.gif','Acre.gif','rodape.png');
+	PreloadImages('../imagens/logo.png','tocantins.gif','rondonia.gif','acre.gif','rodape.png');
 
 
 	SetUpItems(ShuffleQs,QsToShow);
