--- atividades/0104GEO001/0104GEO00107.js	2012-11-09 15:52:03.266149476 -0200
+++ atividades/0104GEO001/0104GEO00107.js.new	2012-11-09 16:07:37.246163510 -0200
@@ -445,7 +445,7 @@
 
 
 
-	PreloadImages('../imagens/logo.png','Parana.gif','Santa_Catarina.gif','Rio_Grande_do_Sul.gif','rodape.png');
+	PreloadImages('../imagens/logo.png','parana.gif','santa_catarina.gif','rio_grande_do_sul.gif','rodape.png');
 
 
 	SetUpItems(ShuffleQs,QsToShow);
