--- atividades/0602HIS092/0602HIS09201.js	2012-11-09 16:07:45.610163634 -0200
+++ atividades/0602HIS092/0602HIS09201.js.new	2012-11-09 16:07:45.610163634 -0200
@@ -791,7 +791,7 @@
 }
 
 
-function clicar() { //função responsável pelo tratamento do botão direito do mouse no IE
+function clicar() { //fun&ccedil;&atilde;o respons&aacute;vel pelo tratamento do bot&atilde;o direito do mouse no IE
       		if (event.button==2) {
         		alert(' Obrigado pela Visita! ');
       		}
