--- atividades/0802POR001/0802POR00101.js	2012-11-09 15:51:57.234149385 -0200
+++ atividades/0802POR001/0802POR00101.js.new	2012-11-09 16:08:02.734163893 -0200
@@ -507,7 +507,7 @@
 
 
 
-	PreloadImages('imagens/ok.gif','imagens/no.gif','imagens/unha.gif','imagens/banho.jpg','imagens/joaninha.jpg','imagens/minhoca.jpg','imagens/ninho.jpg','imagens/galinha.JPG','imagens/caminhao.png');
+	PreloadImages('imagens/ok.gif','imagens/no.gif','imagens/unha.gif','imagens/banho.jpg','imagens/joaninha.jpg','imagens/minhoca.jpg','imagens/ninho.jpg','imagens/galinha.jpg','imagens/caminhao.png');
 
 
 
