--- atividades/0802POR001/0802POR00101.js	2012-10-02 09:57:53.087845260 -0300
+++ atividades/0802POR001/0802POR00101.js.new	2012-10-02 10:13:25.215859266 -0300
@@ -507,7 +507,7 @@
 
 
 
-	PreloadImages('imagens/ok.gif','imagens/no.gif','imagens/unha.gif','imagens/banho.jpg','imagens/joaninha.jpg','imagens/minhoca.jpg','imagens/ninho.jpg','imagens/galinha.JPG','imagens/caminhao.png');
+	PreloadImages('imagens/ok.gif','imagens/no.gif','imagens/unha.gif','imagens/banho.jpg','imagens/joaninha.jpg','imagens/minhoca.jpg','imagens/ninho.jpg','imagens/galinha.jpg','imagens/caminhao.png');
 
 
 
