--- atividades/0803POR015/0803POR01502.js	2012-11-09 16:08:29.706164298 -0200
+++ atividades/0803POR015/0803POR01502.js.new	2012-11-09 16:08:29.710164298 -0200
@@ -481,11 +481,11 @@
 }
 
 var Feedback = '';
-var Correct = 'Parab&#x00E9;ns!';
-var Incorrect = 'Algumas respostas est&#x00E3;o incorretas. Tente outra vez!'; 
+var Correct = 'Parab&eacute;ns!';
+var Incorrect = 'Algumas respostas est&atilde;o incorretas. Tente outra vez!'; 
 var GiveHint = 'Foi adicionada a seguinte letra:';
 var CaseSensitive = false;
-var YourScoreIs = 'Sua pontua&#x00E7;&#x00E3;o &#x00E9;: ';
+var YourScoreIs = 'Sua pontua&ccedil;&atilde;o &eacute;: ';
 var Finished = false;
 var Locked = false;
 var Score = 0;
