--- atividades/1002POR004/1002POR00408.js	2012-11-09 15:51:52.230149310 -0200
+++ atividades/1002POR004/1002POR00408.js.new	2012-11-09 16:09:21.910165083 -0200
@@ -602,7 +602,7 @@
 
 
 
-	PreloadImages('Imagens/boca.gif','Imagens/moeda.jpg','Imagens/cadeado.jpg','Imagens/bolo.jpg','Imagens/lobo.gif','Imagens/oculos.gif','Imagens/bebe.gif','Imagens/cabide.jpg','Imagens/balao.jpg','Imagens/abacate.jpg','Imagens/bau.jpg');
+	PreloadImages('imagens/boca.gif','imagens/moeda.jpg','imagens/cadeado.jpg','imagens/bolo.jpg','imagens/lobo.gif','imagens/oculos.gif','imagens/bebe.gif','imagens/cabide.jpg','imagens/balao.jpg','imagens/abacate.jpg','imagens/bau.jpg');
 
 
 	var i = 0;
