--- atividades/1003POR021/1003POR02101.js	2012-11-09 16:10:12.474165841 -0200
+++ atividades/1003POR021/1003POR02101.js.new	2012-11-09 16:10:12.478165842 -0200
@@ -325,7 +325,7 @@
 	}
 	return InArray;
 }
-<!-- Orientadores de Informática - Comentado para que não exibe o resultado no enunciado da atividade -->
+<!-- Orientadores de Inform&aacute;tica - Comentado para que n&atilde;o exibe o resultado no enunciado da atividade -->
 function WriteToInstructions(Feedback) {
 	<!-- document.getElementById('InstructionsDiv').innerHTML = Feedback; -->
 
@@ -441,7 +441,7 @@
 		Frm.submit();
 	}
 
-  	//Orientadores de Informática - Código que exibir o próximo arquivo
+  	//Orientadores de Inform&aacute;tica - C&oacute;digo que exibir o pr&oacute;ximo arquivo
 	parent.location.href="1003POR02102.htm";
 
 }
@@ -797,7 +797,7 @@
 		Frm.submit();
 	}
 
-  	//Orientadores de Informática - Código que exibir o próximo arquivo
+  	//Orientadores de Inform&aacute;tica - C&oacute;digo que exibir o pr&oacute;ximo arquivo
 	parent.location.href="1003POR02102.htm";
 }
 
